pom.xml

```diff
@@ -60,61 +41,40 @@
         <dependency> <!-- Plan API -->
             <groupId>com.djrapitops</groupId>
             <artifactId>plan-api</artifactId>
-            <version>5.2-R0.1</version>
+            <version>5.2-R0.9</version>
             <scope>provided</scope>
         </dependency>
 
         <!-- Plugin API Dependency -->
         <dependency>
             <groupId>com.github.bloodmc</groupId>
-            <artifactId>GriefDefenderAPI</artifactId>
-            <version>2663329cb5</version>
+            <artifactId>GriefDefenderApi</artifactId>
+            <version>920a610</version>
             <scope>provided</scope>
         </dependency>
-
-        <!-- Platforms -->
-
+        <!-- Needed by GD -->
         <dependency>
-            <groupId>org.spigotmc</groupId>
-            <artifactId>spigot-api</artifactId>
-            <version>1.12.2-R0.1-SNAPSHOT</version>
-            <scope>provided</scope>
+            <groupId>com.flowpowered</groupId>
+            <artifactId>flow-math</artifactId>
+            <version>1.0.3</version>
         </dependency>
```

Comment:
This dependency is provided by GriefDefenderAPI, so you don't have to declare it (also, when the <scope> for a dependency isn't specified, Maven defaults to compile, which shades the dependency and might cause issues later).